resources/assets/js/components/ApplicantProfile/ProfileBasicInformation.tsx

```diff
@@ -0,0 +1,219 @@
+          <p>
+            To change these go to:{" "}
+            <a data-c-color="c1" href="#">
+              Account Settings
+            </a>
+          </p>
+        </div>
+        <div data-c-input="select">
+          <label htmlFor="SEL2">{intl.formatMessage(myBasicInformationMessages.citizenStatus)}</label>
+          <span>Required</span>
+          <div data-c-grid-item="base(1of3)">
+            <i className="fas fa-caret-down" />
+            <select required id="SEL2">
+              <option disabled selected>
+                Select a status...
+              </option>
+              <option>Canadian Citizen</option>
+              <option>Not a citizen</option>
+            </select>
+          </div>
+          <span>{intl.formatMessage(myBasicInformationMessages.inputEreror)}</span>
+        </div>
+        <div data-c-input="select">
+          <label htmlFor="SEL2">
+            {intl.formatMessage(myBasicInformationMessages.isVerteran)}
+          </label>
+          <span>Required</span>
+          <div data-c-grid-item="base(1of3)">
+            <i className="fas fa-caret-down" />
+            <select required id="SEL2">
+              <option>No</option>
+              <option>Yes</option>
+            </select>
+          </div>
+          <span>{intl.formatMessage(myBasicInformationMessages.inputEreror)}</span>
+        </div>
+        <h2 data-c-heading="h2" data-c-margin="bottom(1)">
+          {intl.formatMessage(myBasicInformationMessages.heading)}
+        </h2>
+
+        <div data-c-grid-item="base(1of3)" data-c-input="radio">
+          <label htmlFor="RG2">
+            {intl.formatMessage(myBasicInformationMessages.isGCEmployee)}
+          </label>
+          <span>Required</span>
+          <div id="RG2" role="radiogroup">
+            <label htmlFor="rB1">
+              <input id="rB1" required name="radioB" type="radio" />
+              <span>Current GC Employee</span>
+            </label>
+            <label htmlFor="rB2">
+              <input id="rB2" required name="radioB" type="radio" />
+              <span>Not a GC employee</span>
+            </label>
+          </div>
+          <span>{intl.formatMessage(myBasicInformationMessages.inputEreror)}</span>
+        </div>
+
+        <label htmlFor="SEL2">Current classification and level</label>
+        <ClassificationDropdowns gocClassifications={gocClassifications} />
+
+        <label htmlFor="SEL2">Add previous Government classifications</label>
+        <div id="list-previous-gov-class">
+          <ol>
+            <li>
+              {/* Turn this into a reuseable component */}
+              <div data-c-grid="gutter top">
+                <ClassificationDropdowns gocClassifications={gocClassifications} data-c-grid-item="base(2of2) tl(2of3)" />
+                <div
+                  data-c-grid-item="base(1of1) tl(1of3)"
+                  data-c-input="select"
+                >
+                  <a href="#" data-c-color="c1">
```

Comment:
Avoid using anchor elements for something that should be a button, regardless of how the mockup looks 😄 
For deleting an element from a list, we usually use a trash icon:
```
<button>
   <i className="fa fa-trash" />
</button>